scripts/ci/lint-all-commits

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
+
+GIT_BRANCH="${env.BRANCH_NAME}"
```

Comment:
@juliangieseke env.BRANCH_NAME is the jenkins naming, you would need to do `GIT_BRANCH=env.BRANCH_NAME ./scripts/ci/lint-all-commits` in the Jenkinsfile